ChildProject/pipelines/eafbuilder.py

```diff
@@ -19,15 +21,24 @@ def create_eaf(
     contxt_on: int,
     contxt_off: int,
     template: str,
+    speech_segments: pd.DataFrame = None,
+    imported_set: str = None,
+    imported_format: str = None,
 ):
     import pympi
 
     eaf = pympi.Elan.Eaf(etf_path)
 
     ling_type = "transcription"
+    """
     eaf.add_tier("code_" + eaf_type, ling=ling_type)
     eaf.add_tier("context_" + eaf_type, ling=ling_type)
     eaf.add_tier("code_num_" + eaf_type, ling=ling_type)
+    """
```

Comment:
this is a bit where my very poor read-only skills in python are tricking me... what is the """? and why are lines 34-37 so similar to 39-40, that look like a more general version of 34-36?

Reply:
I also wonder about the ling_type parameter...

Reply:
""" surrounded blocks can be considered as comments.
Here, we thought it was better to have those three eaf tiers being children of a supertier 'SAMPLER' . This was used for the lyon-2022 annotation campaign for example. Not sure how definitive we want this behaviour to be.